lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivity.tsx

```diff
@@ -16,6 +16,8 @@ export type OrganizationActivityProps = {
   organizationPublicId: string;
 };
 
+const buildCourseHref = (id: number) => urlPath`/courses/${String(id)}`;
```

Comment:
I probably would have used `assignmentURL` and `courseURL` as the names for helpers like this. An `href` target is just one use for a URL.

Reply:
Yep, makes sense.